azblob/user_delegation_credential.go

```diff
@@ -0,0 +1,39 @@
+package azblob
+
+import (
```

Comment:
Should this be a zc_ file? Not sure if we can make a user delegation sas for queues and files. The rest docs don't seem to be updated

Reply:
From my understanding, User Delegation SAS is blob-exclusive at the moment and Ze doesn't think it'll be coming to files/ADLSG2.